google-datacatalog-qlik-connector/src/google/datacatalog_connectors/qlik/prepare/datacatalog_entry_factory.py

```diff
@@ -72,6 +72,53 @@ def make_entry_for_app(self, app_metadata):
 
         return generated_id, entry
 
+    def make_entry_for_custom_property_definition(
+            self, custom_property_def_metadata):
+
+        entry = datacatalog.Entry()
+
+        generated_id = self.__format_id(
+            constants.ENTRY_ID_PART_CUSTOM_PROPERTY_DEFINITION,
+            custom_property_def_metadata.get('id'))
+        entry.name = datacatalog.DataCatalogClient.entry_path(
+            self.__project_id, self.__location_id, self.__entry_group_id,
+            generated_id)
+
+        entry.user_specified_system = self.__user_specified_system
+        entry.user_specified_type = \
+            constants.USER_SPECIFIED_TYPE_CUSTOM_PROPERTY_DEFINITION
+
+        entry.display_name = self._format_display_name(
+            custom_property_def_metadata.get("name"))
+        entry.description = custom_property_def_metadata.get("description")
+
+        # The linked_resource field is not fulfilled because there is no way to
```

Comment:
Does the linked_resource need to be clickable?
In the RDBMS connectors we use it as [URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier). It is a way to uniquely identify the resource address, but it does not need to be clickable through the web.

You can combine the tag fields you are ingesting, to find out where the custom property is located right? Why not build the linkedResource from there?

I suggest that the linkedResource should be leveraged to uniquely identify the resource, and avoid the overhead of combining multiple fields to do that.

WDYT?

Reply:
All other assets managed by the Qlik connector have valid URLs as their linked resources. It's possible to generate a URI for the custom property, but we would introduce a mixed behavior for this field by doing so, which I see as a bad thing. By the way, even the URIs are displayed as HTTP links in the Data Catalog UI, and generating linked resources that do not resolve to a valid address might be confusing for the users. WDYT? 🤔